docs/source/installation.rst

```diff
@@ -27,68 +27,63 @@ The following table includes the platforms that TornadoVM can be executed.
-+---------------------------+-----------------------------------------------------------+-----------------+----------------------+
-| macOS Catalina 10.15.3    | OpenCL for GPUs and CPUs >= 1.2                           | Not supported   | Not supported        |
-+---------------------------+-----------------------------------------------------------+-----------------+----------------------+
-| macOS Big Sur 11.5.1      | OpenCL for GPUs and CPUs >= 1.2                           | Not supported   | Not supported        |
-+---------------------------+-----------------------------------------------------------+-----------------+----------------------+
-| macOS Sonoma 14.3         | OpenCL for GPUs and CPUs >= 1.2                           | Not supported   | Not supported        |
-+---------------------------+-----------------------------------------------------------+-----------------+----------------------+
-| Apple M1                  | OpenCL for GPUs and CPUs >= 1.2                           | Not supported   | Not supported        |
+| Apple M1/M2/M3            | OpenCL for GPUs and CPUs >= 1.2                           | Not supported   | Not supported        |
 +---------------------------+-----------------------------------------------------------+-----------------+----------------------+
 | Windows 10/11             | OpenCL for GPUs and CPUs >= 2.1, FPGAs not tested         | CUDA 12.0+      | Level-Zero >= 1.2    |
 +---------------------------+-----------------------------------------------------------+-----------------+----------------------+
+| Windows WSL               | OpenCL for GPUs and CPUs >= 2.1, FPGAs not tested         | Not Supported   | Level-Zero >= 1.2    |
++---------------------------+-----------------------------------------------------------+-----------------+----------------------+
 
-**Note:** The SPIR-V backend of TornadoVM is only supported for Linux OS.
-Besides, the SPIR-V backend with Level Zero runs on Intel HD Graphics (integrated GPUs), and Intel ARC GPUs.
+**Note:** The SPIR-V backend of TornadoVM is supported for Linux and Windows.
+The SPIR-V backend with Level Zero runs on Intel HD Graphics (integrated GPUs), and Intel ARC GPUs.
 
 .. _installation:
 
 Installation
 *****************
 
 TornadoVM can be built with three compiler backends and is able to generate OpenCL, PTX and SPIR-V code.
-There are two ways to install TornadoVM:
 
-A) Automatic Installation
-===========================
+Installer Script
+=================
 
-The ``tornadovm-installer`` script provided in this repository will compile/download OpenJDK, CMake, Maven and it will build TornadoVM.
+The ``tornadovm-installer`` script provided in this repository will compile/download ``OpenJDK``, ``CMake``, ``Maven`` and it will build the TornadoVM project.
 This installation script has been tested on Linux, macOS and Windows.
```

Comment:
If you don't apply the suggestion, consider changing the "installation script" to "installer script" for consistency.